open-company/open-company-web
# Back to Fullstory

### Review with https://github.com/belucid/open-company-infrastructure/pull/91

To test:
- deploy to staging (probably already there)
- open a session w/o adblockers
- [x] check if you see FullStory bundlers being sent
- go to FullStory site
- [x] check if you see your session
- [x] check if you see your user user-id, name and email
- [x] check if you see your user org name and slug (in user vars)
